tools/internal_ci/linux/grpc_bazel_privileged_docker.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright 2017 gRPC authors.
```

Comment:
Update license to "Copyright 2019 The gRPC Authors" for new files.

Reply:
Thanks, fixed.